src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
```

Comment:
You can add this part of code - days * 40 - into variable. But I think it's not a mistake, just code style)